static/sass/patterns/_sections.scss

```diff
@@ -6,23 +6,70 @@
   margin: size('gutter') auto;
   max-width: size('full-content');
   padding: 0 size('gutter');
-
-  & ~ & {
-    @include before('') {
-      border-top: size('border-thin') solid color('border-light');
-      display: block;
-      margin: size('double-gutter') 0;
-    }
-  }
 }
 
 .grid-template {
   max-width: 100%;
 }
 
-.separator {
+
+// Dividers
+// --------
+
+[divider] {
+  position: relative;
+  z-index: 5;
```

Comment:
should we add a z-index list to this site?

Reply:
new story…